CONTRIBUTING.md

```diff
@@ -78,8 +78,16 @@ Finally, as with the earlier directions, when finished exit by pressing ctrl+c o
 7. Save.
 8. Set IDE key to `ROOT_SERVER_DEBUG`.
 9. Save.
-10. Turn on remote debugging by press the button in the toolbar. ![image1](img/3.png)
+10. Turn on remote debugging by press the button in the toolbar. ![image1](docker/img/3.png)
 11. Set any breakpoints, and the code should pause there.
 
 ![image1](docker/img/1.png)
+
 ![image2](docker/img/2.png)
+
+## Debugging code that uses packages in `vendor`
+xdebug may have problems if there are missing source files. So if you encounter an error when running with the debugger
```

Comment:
You shouldn’t need to do this - there should be a way to use compose to install all of the dependencies for you. I think Patrick mentioned a command, but have a look at the travis yaml, and it should have the relevant commands.